src/app/config/config.c

```diff
@@ -5735,14 +5735,14 @@ options_init_logs(const or_options_t *old_options, or_options_t *options,
       }
 
       if (!strcasecmp(smartlist_get(elts, 0), "android")) {
-#ifdef HAVE_ANDROID_LOG_H
+#ifdef __ANDROID__
```

Comment:
Since we're no longer using HAVE_ANDROID_LOG_H, let's remove the code to generate it from configure.ac.

Reply:
This pull request does remove all traces that I could find.  What did I miss?

Reply:
Never mind, you're right. I missed the part where you removed `AC_CHECK_HEADERS`